src/lexer/src_string.sig

```diff
@@ -1,7 +1,9 @@
 signature SRC_STRING =
 sig
     type yypos = int
-
+    exception StringNotClosed of int * int (* line position * line number *)
+    val startPos : int ref
+    val buildingString : bool ref
```

Comment:
Can we make this `val closed : unit -> bool` to be consistent with `SrcComment`?